src/test/java/database/service/JsonDatabaseServiceTest.java

```diff
@@ -171,9 +171,9 @@ public void getByFiltersAllFiltersMatchTest() {
 
     @Test
     public void getByFiltersOnlyOneFilterMatchesTest() {
-        List<Student> students = List.of(thirdStudent);
+        List<Student> students = List.of();
```

Comment:
This test is designed for one matched filter, but in fact filter doesn't return any student. Please fix.